src/Controller/SearchController.php

```diff
@@ -2,16 +2,39 @@
 
 namespace App\Controller;
 
+use App\Repository\SearchesRepository;
 use App\Service\OxfordDictionary\OxfordDictionary;
 use App\Service\OxfordDictionary\OxfordDictionaryException;
 use App\Service\SearchesService;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class SearchController extends AbstractController
 {
+    /**
+     * Returns array of possible words with same beginning
+     *
+     * @param Request $request
+     * @param SearchesRepository $search
+     * @return JsonResponse
+     */
+    #[Route('/quick_search', name: 'quick_search')]
+    public function quickSearch(Request $request, SearchesRepository $search)
+    {
+        $wordBegin = $request->query->get('wordBegin');
+        if(strlen($wordBegin)>2){
+            return  new JsonResponse([
+                'status'    => 1,
+                'matches'   => array_column((array) $search->getFastSearchMatchWords($wordBegin), 'word')
```

Comment:
Make a function name shorter.
Function should return array with words, so remove array_column function and type casting.

Reply:
Function name fully represent what it does - so I believe it's ok.
Type casting remove - ok
array_column should be here, it's a part of the algorithm.